docs/osf-trivia-list/dasharo.md

```diff
@@ -126,3 +126,26 @@ applications.
 [cb-distro]: https://doc.coreboot.org/distributions.html
 [intel-quark]: https://mail.coreboot.org/hyperkitty/list/coreboot@coreboot.org/thread/YRJQIPVK5WHACT64TH42CLGD4TXG3XTS/#PZUIFZZHRK7M3NLBNLI6VUBD4O52245B
 [vpub]: https://vpub.dasharo.com/
+
+## What is Dasharo binary blob policy?
+
+Modern x86 platforms' firmware requires closed source blobs to be integrated
+into the image to properly initialize the silicon. The ecosystem is shifting
+towards designs and technologies with a lot of small microcontrollers and
+intellectual property (IP) blocks specialized in a very thin range of tasks.
+Those microcontrollers and IP blocks typically require firmware blobs as well.
+Some of the blobs are clearly visible, some may be obfuscated and hidden inside
+the silicon or other firmware blobs (e.g. Intel Management Engine region
+contains multiple other blobs besides the ME firmware -
+[more about Intel ME blob](me.md)).
+
+So Dasharo's binary blob policy is as follows:
+
+> Integrate only the necessary amount of blobs required for proper platform
+> operation and minimize the amount of blobs that are optional whenever
+> possible by providing open equivalent implementations or removing them if
+> there is no functional impact on the platform operation.
```

Comment:
I guess we would like to attest binary blobs and document those properly to maximize trustworthiness.

Reply:
@miczyg1 yes, that would be good enough. For now. Maybe link to GH issues regarding that?

Reply:
I can see the issue is there, so it seems it can be resolved. Please object @pietrushnic when necessary and we can fix in a followup MR.